samples/IndexManagement/Program.py

```diff
@@ -0,0 +1,613 @@
+        print("\n" + "-" * 25 + "\n6. Collection created with index policy")
+        print_dictionary_items(created_collection["indexingPolicy"])
+
+        collection_link = GetCollectionLink(database_id, COLLECTION_ID)
+        doc1 = client.CreateDocument(collection_link, { "id" : "doc1", "region" : "USA" })
+        doc2 = client.CreateDocument(collection_link, { "id" : "doc2", "region" : "UK" })
+        doc3 = client.CreateDocument(collection_link, { "id" : "doc3", "region" : "Armenia" })
+        doc4 = client.CreateDocument(collection_link, { "id" : "doc4", "region" : "Egypt" })
+
+        # Now ordering against region is allowed. You can run the following query
+        query = { "query" : "SELECT * FROM r ORDER BY r.region" }
+        message = "Documents ordered by region"
+        QueryDocumentsWithCustomQuery(client, collection_link, query, message)
+
+        # You can also perform filters against string comparison like >= 'UK'. Note that you can perform a prefix query, 
+        # the equivalent of LIKE 'U%' (is >= 'U' AND < 'U')
+        query = { "query" : "SELECT * FROM r WHERE r.region >= 'U'" }
+        message = "Documents with region begining with U"
+        QueryDocumentsWithCustomQuery(client, collection_link, query, message)
+
+        # Cleanup
+        client.DeleteCollection(collection_link)
+        print("\n")
+    except errors.DocumentDBError as e:
+        if e.status_code == 409:
+            print("Entity already exists")
+        elif e.status_code == 404:
+            print("Entity doesn't exist")
+        else:
+            raise
+
+def PerformIndexTransformations(client, database_id):
+    try:
+        DeleteCollectionIfExists(client, database_id, COLLECTION_ID)
+        database_link = GetDatabaseLink(database_id)
+        # collections = Query_Entities(client, 'collection', parent_link = database_link)
+        # print(collections)
+
+        # Create a collection with default indexing policy
+        created_collection = client.CreateCollection(database_link, {"id" : COLLECTION_ID})
+        print(created_collection)
+
+        print("\n" + "-" * 25 + "\n7. Collection created with index policy")
+        print_dictionary_items(created_collection["indexingPolicy"])
+
+        # Insert some documents
+        collection_link = GetCollectionLink(database_id, COLLECTION_ID)
+        doc1 = client.CreateDocument(collection_link, { "id" : "dyn1", "length" : 10, "width" : 5, "height" : 15 })
+        doc2 = client.CreateDocument(collection_link, { "id" : "dyn2", "length" : 7, "width" : 15 })
+        doc3 = client.CreateDocument(collection_link, { "id" : "dyn3", "length" : 2 })
+        print("Three docs created with ids : ", doc1["id"], doc2["id"], doc3["id"])
+
+        # Switch to lazy indexing and wait till complete.
+        print("Chaning the indexing mode of the collection from default to lazy")
+        created_collection['indexingPolicy']['indexingMode'] = documents.IndexingMode.Lazy
```

Comment:
please do not add lazy indexing sample. (remove)

Reply:
Removed this part which was demonstrating the use of Lazy indexing. Now I only have one line in the beginning of the file (description) "*indexingMode can be either of consistent, lazy or none*"